src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -402,3 +418,25 @@ export default function EnvironmentEditor({
         </div>
     );
 }
+
+/*
+<div className={classes.emptyButtonsContainer}>
+                        <div className={classes.buttonContainer}>
+                            <Button
+                                startIcon={<AddIcon />}
+                                className={classes.button}
+                                onClick={onCreateButtonClick}
+                            >
+                                New Scene from Scratch
+                            </Button>
+                        </div>
+                        <div className={classes.buttonContainer}>
+                            <Button
+                                startIcon={<AccountBalanceIcon />}
+                                className={classes.button}
+                            >
+                                New Scene from Template
+                            </Button>
+                        </div>
+                    </div>
+*/
```

Comment:
Make sure to clean up PRs before requesting review. I'll do the change myself to save some back-and-forth time but please be mindful of this next time!